pipeline/test/elasticsearch-helpers.test.ts

```diff
@@ -52,3 +58,32 @@ describe("ensureIndexExists", () => {
     });
   });
 });
+
+describe("getParentDocumentIDs", () => {
+  it("queries for batches of potential child document IDs", async () => {
+    const totalDocs = 100;
+    const batchSize = 10;
+
+    const documents = identifiedDocuments(totalDocs);
+    const elasticScrollDocuments = createElasticScrollDocuments(documents);
+    const testClient = {
+      helpers: {
+        scrollDocuments: elasticScrollDocuments,
+      },
+    } as unknown as ElasticClient;
+
+    const finalDoc = await lastValueFrom(
+      range(totalDocs).pipe(
+        map((n) => n.toString()),
+        getParentDocumentIDs(testClient, {
+          index: "test",
+          identifiersField: "childIds",
+          batchSize: batchSize,
+        })
+      )
+    );
+
+    expect(elasticScrollDocuments).toHaveBeenCalledTimes(totalDocs / batchSize);
+    expect(finalDoc).toBe(totalDocs.toString());
```

Comment:
At first glance, I found this a little confusing (though a closer look clarified it)  - why is finalDoc the same value as totalDocs?  Is it picking out a count or an id?

I wonder if it might be clearer if identifiedDocuments were to create an id based on the number, rather than just stringifying it (e.g. "id_100")

Reply:
ah yes, I was being lazy here and I think had lied to myself that I'd come back to it - will sort it